appscale/tools/utils.py

```diff
@@ -120,3 +120,12 @@ def queues_from_xml(contents):
     queues['queue'].append(queue)
 
   return queues
+
+
+def mkdir(dir_path):
+  """ Creates a directory.
+
+  Args:
+    dir_path: The path to create.
+  """
+  return os.system("mkdir -p " + dir_path)
```

Comment:
Wouldn't it be better to use os.mkdir or os.makedirs? We should also check if we get a permissions denied, or already exists as a simple checks.